### DISCUSSION TITLE
Common attributes in HTTP semantic conventions

### DISCUSSION THREAD

**Discussion by plantfansam** [open-telemetry/opentelemetry-specification#2683] [Q&A] [answered]
[The HTTP semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md) include a large section of "[Common Attributes](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md#common-attributes)". Is the intent that these considered for inclusion in both client and server spans? I assumed so, but I did not see this mentioned directly in the spec.

**Comment by arminru** [comment, marked as answer]
@plantfansam Yes, they apply to both. #3044 should clarify this better.

**Reply by plantfansam** [reply]
🙏  thanks!
